token-metadata/program/src/instruction/mod.rs

```diff
@@ -357,6 +384,8 @@ pub enum MetadataInstruction {
     #[account(2, name="edition", desc="Edition")]
     #[account(3, name="mint", desc="Token mint")]
     #[account(4, name="token_program", desc="Token Program")]
+    #[account(5, name="system_program", desc="System program")]
```

Comment:
Does the `edition` account need to be writable?

Reply:
Yes, thanks!